docs/developers_guide/ocean/test_groups/global_ocean.rst

```diff
@@ -454,6 +436,82 @@ The default config options for this mesh are:
 The vertical grid is a ``60layerPHC`` profile (see :ref:`dev_ocean_framework_vertical`)
 with 60 vertical levels ranging in thickness from 10 to 250 m.
 
+.. _dev_ocean_global_ocean_kuroshio:
+
+Kuroshio8to60 and Kuroshio12to60
+++++++++++++++++++++++++++++++++
+
+The ``Kuroshio8to60`` and ``Kuroshio12to60`` mehses are designed to explore
+dynamics of the Kuroshio Current.
+
+The class
+:py:class:`compass.ocean.tests.global_ocean.mesh.kuroshio.KuroshioBaseMesh`
+defines the resolution for the meshes, where the finest resolution comes from
+the ``min_res`` config option in the ``[global_ocean]`` section of the config
+file.
+
+The ``compass.ocean.tests.global_ocean.mesh.kuroshio8to60`` and
+``compass.ocean.tests.global_ocean.mesh.kuroshio12to60`` modules include
+namelist options appropriate for forward simulations with split-explicit (but
+not RK4) time integration on this mesh.  These set the time step and default
+run duration for short runs with this mesh.
+
+Except for ``min_res``, default config options for these meshes come from a
+shared config file in the ``compass.ocean.tests.global_ocean.mesh.kuroshio``
+module:
```

Comment:
@ytakano3, please check this documentation and suggest modifications here if you would like (using the "Add suggestions" button with the +/-).

Reply:
@xylar Minor comment (change) added.